src/compiler/cross-team-collaboration.md

```diff
@@ -0,0 +1,73 @@
+# Cross-team Collaboration
+If you are a member of another team and would like to raise an issue with the
+compiler team..
+
+## ..for discussion
+Write a comment on a GitHub issue describing the reason for the nomination
+(i.e. what decision needs to be made/what opinion is sought; what are the
+relevant parts to the compiler team, etc) and add the `I-compiler-nominated`
+label to a issue (you can use `@rustbot label +I-compiler-nominated` to do
+this).
+
+Once nominated, the issue will be discussed in a upcoming [triage
+meeting](./triage-meeting.html). The compiler team doesn't always get through
+all nominated issues each week, so it can take more than one meeting for your
+issue to be discussed.
+
+Once discussed, a member of the team will comment on the issue with the
+conclusion of the discussion and linking to the relevant Zulip chat.
+
+## ..to be fixed
+If there is an existing working relationship between a member of the requesting
+team and a contributor to the compiler, then the first option that a team has
+for requesting tasks be completed is to ping that contributor and ask if they
+can complete the task. It is recommended that pings take place in public Zulip
+channels so that..
+
+- ..other contributors that have free time have the opportunity to
+  offer their help.
+- ..other compiler team members/leadership can ensure that requests being made
+  are reasonable (see the rest of this section for the types of issues that the
+  compiler team commits to prioritizing on behalf of other teams).
+
+It is worth considering the available bandwidth of the contributor that the
+request is being made of, and whether their areas of expertise in the compiler
+are relevant.
+
+When there is not a appropriate contact in the compiler team to reach out to
+directly, write a comment on a GitHub issue (or create an issue) describing the
+task that needs completed. Teams should nominate issues for the compiler team
+when issues..
+
+- ..are not already tracked by/part of an existing initiative or working group
+  and..
+- ..are blocking/impeding the work of the other team (e.g. a feature or bug
+  preventing the stabilization of something otherwise complete), but..
+- ..aren't absolutely mission-critical - a soundness bug or otherwise critical
+  issue will be prioritized by the [prioritization working
+  group](./prioritization.html) and addressed through the compiler team's other
+  processes for these bugs.
```

Comment:
```suggestion
  processes for these bugs. If the issue in question lacks a priority label, you can add the `I-prioritize` label and it will be enqueue for prioritization.
```

Reply:
(I skimmed the linked document for the prioritization working group, and while it did include this information, I thought it was a little too hard to find for that link alone to suffice here.)

Another topic that I debate about bringing here: What should someone do if they believe an issue has been *misprioritized* (i.e., has an incorrect prioritization label)? At that point they should say so in a comment and tag it as `I-compiler-nominated`, right?

Reply:
If someone notices a misprioritized issue (due to lack of context/understanding of the problem or other reason):

- they can comment in the relevant Zulip topic (each prioritized issue has one) or in the issue itself
- if there is a bit more to discuss than the priority label, probably `I-compiler-nominated` is better suited to get the full team's attention during the meeting
- I guess in other cases it's probably also fine to straight override the priority label (example: the person has good knowledge/experience of the process)

---

All this could actually be spelled out better in the [WG-prioritization guidelines](https://forge.rust-lang.org/compiler/prioritization/procedure.html). I'd take advantage of this document to add a bit of info also there.

Reply:
Fixed.

> All this could actually be spelled out better in the [WG-prioritization guidelines](https://forge.rust-lang.org/compiler/prioritization/procedure.html). I'd take advantage of this document to add a bit of info also there.

I don't know if by this you meant that I could include these details as part of this PR, but I think I'd prefer to leave that for another PR and land this one just with the collaboration policy.

Reply:
absolutely! I will take care of that when this PR will be merged :)